cmd/cluster/none/create.go

```diff
@@ -93,6 +95,7 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 
 	exampleOptions.None = &apifixtures.ExampleNoneOptions{
 		APIServerAddress: opts.NonePlatform.APIServerAddress,
+		AgentBased:       opts.NonePlatform.AgentBased,
```

Comment:
Maybe a nit but we're handling this new agent option inconsistently - elsewhere it's defined as a new platform, but here it's exposed as an option to the None platform.

Right now, that's probably not a big issue, but we've discussed some ideas around combining the agent provider with infrastructure automation (e.g so the agent image can be booted automatically on metal, vmware, etc).

We haven't resolved if/how to enable that yet, but I'm wondering if treating agent as a platform option consistently throughout the code would perhaps reduce friction in future if we decide to persue automating starting the agent image?

Reply:
Yes, I have an action item to adjust that.